### ISSUE TITLE
not working WIN 10 1709

### ISSUE THREAD

**Issue by Comodore125** [tryallthethings/DNS-Swapper#2]
Hey. Loaded at startup. DNS is not getting switched. I do likely have more difficult netorking because I do have some virtual switch as I have HyperV running on my machine.

**Comment by tryallthethings** [comment]
Hey! Can you maybe screenshot / explain your configuration so I can have a look at it?

**Comment by Comodore125** [comment]
damn, that took such a long time. I am doing some big changes to my network. Running tens of meters of cat6a cables. optical fiber om5 32 meter in total. Buying new switches (2) - managed ones. placing turris omnia back into role of router. upgrading vdsl line to full speed, changing ISP. preparing for second WAN on the roof. Ask me again in 25 days or someting. Or I could maybe send it if I remember. I got some problems with turris that turned into read only filesystem. So I lost reason to use dnsswapper as I got pi-hole in LXC.
